server/service/mdm.go

```diff
@@ -2624,13 +2624,13 @@ func (svc *Service) UploadMDMAppleVPPToken(ctx context.Context, token io.ReadSee
 		return ctxerr.Wrap(ctx, err, "validating VPP token with Apple")
 	}
 
-	decodedTokenBytes, err := base64.StdEncoding.DecodeString(string(tokenBytes))
-	if err != nil {
-		return ctxerr.Wrap(ctx, err, "decoding VPP token")
-	}
+	// decodedTokenBytes, err := base64.StdEncoding.DecodeString(string(tokenBytes))
+	// if err != nil {
+	// 	return ctxerr.Wrap(ctx, err, "decoding VPP token")
+	// }
```

Comment:
Did you mean to leave this commented out?

Reply:
nope! late night coding fun lol